packages/cdk8s-cli/README.md

```diff
@@ -32,24 +34,148 @@ $ cd my-fun-little-project
 $ cdk8s init typescript-app
 ```
 
-## import - Import API objects as constructs
+## import
+
+### Selecting import language
+
+You can specify the language of your imports with the `--language` flag.
+
+`cdk8s` CLI
+
+```code
+cdk8s import --language typescript
+```
+
+`cdk8s.yaml` Config
+
+```yaml
+language: typescript
+```
+
+Both `typescript` and `python` are supported right now, unless noted otherwise.
+
+Example output for `typescript` imports is:
+
+```
+imports/
+  k8s.ts
+```
+
+Example output for `python` imports is:
+
+```
+imports/
+  k8s/
```

Comment:
That feels redundant. I think the import example code should be how to use the import in each language